test/unit/transaction.test.js

```diff
@@ -1285,7 +1367,7 @@ describe('Transaction', function () {
           ty: 'Mobile',
           ac: trustedAccountKey,
           ap: '51424',
-          id: '5f474d64b9cc9b2a',
+          id: '5f4n74d64b9cc9b2a',
```

Comment:
It looks like this id value has changed.

Reply:
Whoa! I know how that happened. Emacs-induced typo. :( `C-a` means "beginning of line" so sometimes I typo and leave a stray "a" somewhere.

Reply:
Though this one looks like an inserted `n`, I'd think.

Reply:
`C-n` is next line. :P (down arrow)

Reply:
Ah, that'd explain it, yeah.